pymagicc/io.py

```diff
@@ -1432,6 +1471,10 @@ class _RadiativeForcingInWriter(_Writer):
     pass
 
 
+class _SurfaceTemperatureInWriter(_Writer):
+    pass
```

Comment:
Should that be NotImplementedError or something?

Reply:
Nope. The `pass` just means do nothing i.e. `_SurfaceTemperatureInWriter` is identical to `_Writer`. I added the extra classes just to make the namespace clearer and for the regex name matching stuff later on.

Reply:
Haha, read the code (that is me :-)